xml/networking-octavia_admin.xml

```diff
@@ -403,6 +403,145 @@ Updated port: 0b0301b9-4ee8-4fb6-a47c-2690594173f4</screen>
    </para>
   </warning>
  </section>
+ <section xml:id="octavia-admin-delete">
+   <title>Removing load balancers</title>
+   <para>
+     You may find that you are unable to delete load balancers that are not
+     in the <literal>ACTIVE</literal> state. If you have load balancers that
+     cannot be deleted but are in the <literal>ERROR</literal> or
+     <literal>Pending_delete</literal> state, follow these directions when you
+     want to delete them. To clean up the bad instances run the
+     following MySQL commands manually.
```

Comment:
Suggested rewrite of this para - in technical writing, it is best practice to avoid saying things like "may" as people inevitably skip over what is considered an unknown. Suggestion:

`The following instruction details how to delete a load balancer that is in the <literal>ERROR</literal> or <literal>PENDING_DELETE</literal> state. The following commands need to be executed manually to clean up the erroring instances:`

Reply:
Fixed